homeassistant/components/meteo_france/__init__.py

```diff
@@ -83,7 +84,13 @@ async def _async_update_data_alert() -> CurrentPhenomenons:
         update_method=_async_update_data_rain,
         update_interval=SCAN_INTERVAL_RAIN,
     )
-    await coordinator_rain.async_config_entry_first_refresh()
+    try:
+        await coordinator_rain._async_refresh(log_failures=False)  # noqa: SLF001
```

Comment:
why do we use `_async_refresh`? And not a public function?

Reply:
As written in the description: to only warn but not raise an error.

Otherwise the user will both have an error and a warn.

Because `async_refresh` log_failures=True !

Reply:
Oh It didn't dawn on me that that was what you meant with that